### PR TITLE
Use eslint-config-stripes

### PR DESCRIPTION
**Pull request by cherewaty**
Switches to using shared FOLIO eslint config: https://github.com/folio-org/eslint-config-stripes

### CODE REVIEW

File: src/Root.js [hunk 1]
```diff
@@ -48,9 +48,8 @@ class Root extends Component {
       this.epics[key] = epic;
       this.props.epics.add(epic);
       return true;
-    } else {
-      return false;
     }
+    return false;
```
Comment (cowboyd) [inline src/Root.js]: Is this a linting rule that got triggered?

Comment (cherewaty) [reply src/Root.js]: Yes https://eslint.org/docs/rules/no-else-return
I have mixed feelings about that rule.

**Comment by cherewaty** [comment]
Re: Jenkins build failures: is it running into a cached npm registry or something? `debug`, a dependency of `finalhandler` (a dependency of `express`), had a `2.6.9` release 8 hours ago that `npm install` on Jenkins can't find.

File: src/Root.js (hunk 1 above)
Comment (cowboyd) [reply src/Root.js]: Yeah, these days I favor symmetry over efficiency when given a choice

Comment (MikeTaylor) [reply src/Root.js]: As a general policy, we want ESLint to be our servant, not our master. Bondage-and-discipline rules like no-nested-ternary and no-plusplus can go, with no period of mourning required. no-else-return definitely belongs in this category!

**Comment by funkymalc** [comment]
Right.  npmjs.org is cached.    The builds passes now.

File: src/Root.js (hunk 1 above)
Comment (cherewaty) [reply src/Root.js]: https://github.com/folio-org/eslint-config-stripes/pull/3

**Pull request merged**